notebooks/data-sources/TestGrid/metrics/blocked_timed_out.ipynb

```diff
@@ -2,67 +2,126 @@
  "cells": [
```

Comment:
<p>timestamp has already been defined above, so can be removed here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>